recipes/paraview/5.13.2/zen2/compilers.yaml

```
@@ -0,0 +1,5 @@
+bootstrap:
+  spec: gcc@11
```

is there any legit reason why we bootstrap `gcc@12` with `gcc@11`?

```suggestion
  spec: gcc@12
```

---

it couold be updated, it was there since the original 2023 PR commits, if we change it, we ought to rerun tests etc etc -- can we just leave it alone.

---

aha - that's the eiger build only - the gh200 build was updated with gcc12, but I vaguely remember that eiger needed 11 - I will update it and trigger builds and see if all works.